Freewheel/BasicFreewheelPlayer/objc/BasicFreewheelPlayer/ViewController.m

```diff
@@ -60,24 +60,16 @@ - (void)setup
 
     BCOVPlayerSDKManager *manager = [BCOVPlayerSDKManager sharedManager];
 
-    _playbackController = [manager createFWPlaybackControllerWithAdContextPolicy:[self adContextPolicy] viewStrategy:nil];
```

Comment:
@jblaker perhaps we can leave a comment about this method rather than removing it completely.  You can still say // Creating a playback controller based on this code will initialize a Freewheel component using its default settings.     // See BCOVFWSessionProvider.h for details.